intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -402,9 +484,43 @@ \subsection{Connect}
 
 %% TODO: how to fix this on windows/putty? generally this is not an issue I believe
 \ifmacORlinux
+\ifantwerpen
 Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifbrussel
+Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifgent
+Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifleuven
+Open the .bashrc or the .profile on your local machine with your
```

Comment:
I guess if you want to fix this, it should be specified in bashrc, but this issue is generally only seen in mac OS X, where .bash_profile is always executed, also on non login shells, so this always fixed it. bashrc is better thought